BigQueryConnection/VERSION

```diff
@@ -0,0 +1 @@
+master
```

Comment:
IIRC these need to be 0.0.0 for the tooling to process correctly at the moment.

Reply:
fix't.